### PR TITLE
NIFI-3390 Added support for LDAP HA

### PR DESCRIPTION
**Pull request by pvillard31**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [X] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review..

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/login-identity-providers.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         'Connect Timeout' - Duration of connect timeout. (i.e. 10 secs).
         'Read Timeout' - Duration of read timeout. (i.e. 10 secs).
        
-        'Url' - Url of the LDAP servier (i.e. ldap://<hostname>:<port>).
+        'Url' - Comma separated list of URLs of the LDAP server (i.e. ldap://<hostname>:<port>).
```
Comment (mcgilman) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/login-identity-providers.xml]: I think this documentation update should probably be reflected in administration-guide.adoc too.

Comment (pvillard31) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/login-identity-providers.xml]: You're right, I updated the PR.

**Comment by pvillard31** [comment]
Updated the PR to use a space-separated list of URLs as it seems to be the standard according to http://docs.oracle.com/javase/jndi/tutorial/ldap/misc/url.html

**Comment by mcgilman** [comment]
Thanks @pvillard31! This has been merged to master.

**Pull request closed without merging**